src/Bicep.Core.IntegrationTests/CentralizedProviderVersionManagementTests.cs

```diff
@@ -23,7 +23,7 @@ public class CentralizedProviderVersionManagementTests : TestBase
         public void ProvidersConfig_SupportForConfigManagedProviderDeclarationSyntax_When_ProviderIsBuiltIn(string providerIdentifier, bool shouldSucceed, (string code, DiagnosticLevel level, string message)[] expectedDiagnostics)
         {
             var result = CompilationHelper.Compile(Services, @$"
-                provider {providerIdentifier}
+                extension {providerIdentifier}
```

Comment:
As you mentioned in the PR description, will providerIdentifier be changed in another PR?

Reply:
Yeah, this is tracked by https://github.com/Azure/bicep/issues/14375.